products/linuxmint.md

```diff
@@ -0,0 +1,88 @@
+---
+permalink: /linuxmint
+layout: post
+title: Linux Mint
+command: lsb_release --release
+category: os
+link: https://linuxmint.com/download_all.php
+activeSupportColumn: true
+releaseDateColumn: true
+iconSlug: linuxmint
+sortReleasesBy: "releaseCycle"
+releases:
+  - releaseCycle: "20.2 'Uma'"
+    lts: true
+    release: 2021-07-08
+    support: true
+    eol:     2025-04-01
+    latest: "20.2"
+    link: https://linuxmint.com/edition.php?id=288
+    
+  - releaseCycle: "20.1 'Ulyssa'"
+    lts: true
+    release: 2021-01-08
+    support: false
+    eol:     2025-04-01
+    latest: "20.1"
+    link: https://blog.linuxmint.com/?p=4011
+    
+  - releaseCycle: "20 'Ulyssa'"
+    lts: true
+    release: 2020-06-27
+    support: false
+    eol:     2025-04-01
+    latest: "20"
+    link: https://blog.linuxmint.com/?p=3928
+   
+  - releaseCycle: "19.3 'Tricia'"
+    lts: true
+    release: 2019-12-18
+    support: false
+    eol:     2023-04-01
+    latest: "19.3"
+    link: https://blog.linuxmint.com/?p=3832
+    
+  - releaseCycle: "19.2 'Tina'"
+    lts: true
+    release: 2019-08-02
+    support: false
+    eol:     2023-04-01
+    latest: "19.2"
+    link: https://blog.linuxmint.com/?p=3786
+    
+  - releaseCycle: "19.1 'Tessa'"
+    lts: true
+    release: 2018-12-19
+    support: false
+    eol:     2023-04-01
+    latest: "19.1"
+    link: https://blog.linuxmint.com/?p=3669
+
+  - releaseCycle: "19 'Tara'"
+    lts: true
+    release: 2018-06-29
+    support: false
+    eol:     2023-04-01
+    latest: "19"
+    link: https://blog.linuxmint.com/?p=3597
+
+  - releaseCycle: "18.3 'Sylvia'"
+    lts: true
+    release: 2017-11-27
+    support: false
+    eol:     2021-05-03
+    latest: "18.3"
+    link: https://blog.linuxmint.com/?p=3457
+
+    
+---
+>[Linux Mint](https://linuxmint.com/) is a community-driven Linux distribution based on Ubuntu (which is then based on Debian), bundled with a variety of free and open-source applications. 
```

Comment:
```suggestion
>[Linux Mint](https://linuxmint.com/) is a community-driven Linux distribution for desktop and laptop computers based on Debian and Ubuntu, bundled with a variety of free and open-source applications.
```

Changed wording to reflect upstream is both "Ubuntu and Debian" which is what the project seems to use:

>Linux Mint stands on the shoulder of giants, it is based on Debian and Ubuntu.

Both is important because https://www.linuxmint.com/download_lmde.php exists.

Reply:
Thanks for catching that! Completely missed it.